<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemRandomiser</class>
 <widget class="QDialog" name="ItemRandomiser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item Price Randomiser Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QFormLayout" name="Settings">
     <item row="1" column="0">
      <widget class="QLabel" name="label_MinPrice">
       <property name="text">
        <string>Min Price</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LE_MinPrice">
       <property name="text">
        <string>1</string>
       </property>
       <property name="maxLength">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_MaxPrice">
       <property name="text">
        <string>Max Price</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LE_MaxPrice">
       <property name="text">
        <string>1500000</string>
       </property>
       <property name="maxLength">
        <number>7</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LE_MinPrice</tabstop>
  <tabstop>LE_MaxPrice</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
